doc/optimizers.md

```diff
@@ -1,3 +1,61 @@
+## AdaBelief
+
+*An optimizer for [differentiable separable functions](#differentiable-separable-functions).*
```

Comment:
I don't think this anchor works as expected. Or is this eventually built in a way that works?